gapis/resolve/command_tree.go

```diff
@@ -140,12 +141,40 @@ func CommandTreeNodeForCommand(ctx context.Context, p *path.CommandTreeNodeForCo
 		return nil, fmt.Errorf("Subcommands currently not supported for Command Tree") // TODO: Subcommands
 	}
 
+	ctx = capture.Put(ctx, cmdTree.path.Capture)
+	c, err := capture.Resolve(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	snc, err := SyncData(ctx, cmdTree.path.Capture)
+	if err != nil {
+		return nil, err
+	}
+
 	return &path.CommandTreeNode{
 		Tree:    p.Tree,
-		Indices: cmdTree.indices(api.CmdID(cmdIdx)),
+		Indices: cmdTree.findNodeForCommand(snc, c.Commands, api.CmdID(cmdIdx)),
 	}, nil
 }
 
+func (t *commandTree) findNodeForCommand(snc *sync.Data, cmds []api.Cmd, cmd api.CmdID) []uint64 {
```

Comment:
I'm really not sure about this change. Please can you move this CL into a separate PR?

Reply:
Done.